packages/frontend/src/components/table/TableView.tsx

```diff
@@ -90,15 +95,17 @@ export function TableView<T>({ items, columns, rows }: Props<T>) {
                 {columns.map((column, j) => (
                   <td
                     key={j}
-                    className={cx('h-9 md:h-10', column.minimalWidth && 'w-0')}
+                    className={cx('h-9 md:h-14', column.minimalWidth && 'w-0')}
```

Comment:
maybe row can be a bit higher on mobile, or rosetta smaller, or both

Reply:
I made it smaller, check it out! 
<img width="437" alt="Screenshot 2023-03-09 at 08 52 32" src="https://user-images.githubusercontent.com/38423054/223956363-be36744e-8805-4897-9583-a73621c5ef42.png">